duckduckgo/duckduckgo-privacy-extension
# Update popup UI

**Reviewer:** @jdorweiler 
**CC:** @jagtalon @alohaas 


## Description:



## Steps to test this PR:
- Build and load extension
- Load different sites and check the main popup view and subviews look like in the mockups
- Make [this check](https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/develop/shared/js/ui/templates/top-blocked-truncated.es6.js#L6) always true to quickly test the taller popup with top blocked sites
- Functionality should be unaffected

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
